docs/_includes/events.html

```diff
@@ -0,0 +1,24 @@
+<!-- Events -->
+<section class="page-section" id="events">
+    <div class="container text-center">
+      <h2 class="page-section-heading text-uppercase">Hacktoberfest & GrimoireLab</h2>
+      <div class="row text-left mt-5">
+        <div class="col-lg-5 ml-auto">
+          <p>Help driving growth of open source and make positive contributions to the community! This is how participants from all skill levels can help: </p>
+          <ol>
+            <li class="mb-3"><b>Visit <a href="https://hacktoberfest.digitalocean.com/">hacktoberfest website</a> and sign in with your github account:</b> Once linked, hacktoberfest will send you an email confirmation that you have officially been registered.</li>
+            <li class="mb-3"><b>Choose your favourite <a href="https://github.com/chaoss/grimoirelab/issues">Grimoirelab issue</a> and submitt your Pull Request:</b> issues will be tagged using <button class="button button4">Hacktoberfest</button> label.</li>
```

Comment:
Please, don't use `<button>` element to style a text. That's not a good HTML practice.

Since the website is using bootstrap, you could get desired style using [badge class](https://getbootstrap.com/docs/4.0/components/badge/)

Reply:
thanks for the reference! I'll use badge-dark class then and remove css syle